components/automate-ui/src/app/pages/project/rules/project-rules.component.scss

```diff
@@ -29,6 +29,71 @@ chef-page {
     max-width: 340px;
   }
 
+  /* Custom dropdown */
+  .custom-dropdown {
+    position: relative;
+    vertical-align: middle;
+  }
+
+  .custom-dropdown select {
```

Comment:
@vinay033 If you could nest all the styling for .custom-dropdown
`.custom-dropdown {`
 `   position: relative;`
 `   vertical-align: middle;`
 ` }`

 ` & select {`

Reply:
Thanks, @tarablack01  I glad to see this kind of appreciation from you, this credit goes to @sachin-msys, he has put his efforts for the new styling.